Sim_To_DuT_Interface/DuTLogger/DuTLogger.cpp

```diff
@@ -0,0 +1,339 @@
+//
+// Created by marco on 26.10.21.
+//
+
+#include "DuTLogger.h"
+
+// initialize the logging paths
+std::string DuTLogger::currentLogpathConsole = initializeLoggingPath(LOGGER_TYPE::CONSOLE);
+std::string DuTLogger::currentLogpathData = initializeLoggingPath(LOGGER_TYPE::DATA);
+
+// initialize the static handlers
+quill::Handler* DuTLogger::consoleHandler = DuTLogger::buildConsoleHandler();
+quill::Handler* DuTLogger::consoleFileHandler = DuTLogger::buildFileHandler();
+
+// initialize the static loggers
+quill::Logger* DuTLogger::consoleLogger = DuTLogger::createConsoleLogger("consoleLog", false);
+quill::Logger* DuTLogger::consoleFileLogger = DuTLogger::createConsoleLogger("consoleFileLog", true);
+quill::Logger* DuTLogger::dataLogger = DuTLogger::createDataLogger();
+
+
+/**
+ * Starts the quill engine. Won't start it will the engine is already running.
```

Comment:
The second sentence is strange. I suppose you meant: "Won' start the quill engine again if it is already running."